src/NSaga/SqlServer/IConnectionFactory.cs

```diff
@@ -16,6 +16,12 @@ public interface IConnectionFactory
         /// </summary>
         /// <returns></returns>
         IDbConnection CreateOpenConnection();
+
+        /// <summary>
+        /// Specifies if the connection is to a MySql database.
+        /// </summary>
+        /// <returns></returns>
+        bool ConnectionIsToMySql();
```

Comment:
Breaking change.

Reply:
Also this is implementation problem, not part of the interface contract.

Reply:
And there is only a single implementation of ConnectionFactory and value of `ConnectionIsToMySql()` is always `false`.